charts/spire/values.yaml

```diff
@@ -1,8 +1,11 @@
-# You can enable features that affect all services here.
-# global:
-#   telemetry:
-#     prometheus:
-#       enabled: true
+# You can enable config/features that affect all services here.
+global:
```

Comment:
Do we have to have the extra `global` level or can we just assume if its not under `spire-server`, `spire-agent`, etc. that its global?

Reply:
I think global in the root chart is fine, not sure on the sub-charts though.

- https://helm.sh/docs/chart_template_guide/subcharts_and_globals/#global-chart-values
- https://helm.sh/docs/chart_template_guide/subcharts_and_globals/#helm

Reply:
Removed.

Reply:
> Do we have to have the extra `global` level or can we just assume if its not under `spire-server`, `spire-agent`, etc. that its global?

global is a special value section that helm copies verbatim  into subcharts. no other values can do it. It has to be called global to work.

You can read more about it at:
https://helm.sh/docs/chart_template_guide/subcharts_and_globals/#global-chart-values

Reply:
Thank Kevin, re-reading those globals docs your original with globals in subcharts was probably the better choice. Also considering the fact if we move the subcharts later out to act as a root chart I think using the global config we can maintain a more stable API. Also see my previous suggestion, maybe we should discuss this one in a call on slack to align on the options, pros, and cons. Sorry for this back and forth, also just learning about this.

Reply:
globals section now makes sense to me too.